packages/simple-markdown/src/index.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines */
 /* eslint-disable prefer-spread, no-regex-spaces, guard-for-in, no-console, no-var */
```

Comment:
```suggestion
/* eslint-disable max-lines, prefer-spread, no-regex-spaces, guard-for-in, no-console, no-var */
```
Feel like we should be consistent, either they're all multi line or all one line